qa/mc_sc_forging1.py

```diff
@@ -0,0 +1,211 @@
+#!/usr/bin/env python2
+
+from SidechainTestFramework.sc_test_framework import SidechainTestFramework
+from SidechainTestFramework.sc_boostrap_info import SCNodeConfiguration, SCCreationInfo, MCConnectionInfo, \
+    SCNetworkConfiguration
+from test_framework.util import initialize_chain_clean, start_nodes, \
+    websocket_port_by_mc_node_index, connect_nodes_bi, disconnect_nodes_bi
+from SidechainTestFramework.scutil import bootstrap_sidechain_nodes, start_sc_nodes, generate_next_blocks
+from SidechainTestFramework.sc_forging_util import *
+
+"""
+Check Latus forger behavior for:
```

Comment:
add test for checking self recursive ommers? I.e. fork A have ommers from B, but later B became active, thus B contains ommers from A as well.